[2024-10-17T09:30:57.413+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_from_local_to_postgres.extract_data manual__2024-10-17T09:30:55.272207+00:00 [queued]>
[2024-10-17T09:30:57.427+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_from_local_to_postgres.extract_data manual__2024-10-17T09:30:55.272207+00:00 [queued]>
[2024-10-17T09:30:57.428+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-10-17T09:30:57.458+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): extract_data> on 2024-10-17 09:30:55.272207+00:00
[2024-10-17T09:30:57.466+0000] {standard_task_runner.py:60} INFO - Started process 314 to run task
[2024-10-17T09:30:57.470+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_from_local_to_postgres', 'extract_data', 'manual__2024-10-17T09:30:55.272207+00:00', '--job-id', '651', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp8n94dzgk']
[2024-10-17T09:30:57.471+0000] {standard_task_runner.py:88} INFO - Job 651: Subtask extract_data
[2024-10-17T09:30:57.535+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_from_local_to_postgres.extract_data manual__2024-10-17T09:30:55.272207+00:00 [running]> on host 9c093e78d1a2
[2024-10-17T09:30:57.639+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_from_local_to_postgres' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-17T09:30:55.272207+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-17T09:30:55.272207+00:00'
[2024-10-17T09:30:57.708+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-10-17T09:30:57.803+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl_from_local_to_postgres, task_id=extract_data, execution_date=20241017T093055, start_date=20241017T093057, end_date=20241017T093057
[2024-10-17T09:30:57.843+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-10-17T09:30:57.910+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
